usng4j-api/src/main/java/org/codice/usng4j/UsngCoordinate.java

```diff
@@ -38,6 +38,18 @@
  */
 public interface UsngCoordinate {
 
+  /** RegEx expressions for USNG/MGRS Zone parsing */
+  public final String ZONE_REGEX_STRING = "([1-9]|[1-5][0-9]|60)";
+
+  /** RegEx expressions for USNG/MGRS Latitude Bands parsing */
+  public final String LATITUDE_BAND_REGEX_STRING = "([C-JK-NP-X])\\W?([A-JK-NP-Z][A-JK-NP-V])?";
+
+  /** RegEx expressions for USNG Northing and Easting parsing */
+  public final String USNG_COORDINATE_PART_REGEX_STRING = "(\\W\\d{0,5})?(\\W\\d{0,5})?";
```

Comment:
Should we change ```\\W``` (single non word character) to ```\\s+``` (one or more white space characters) instead? Same comment for other places ```\\W``` is used

Reply:
The scope of this PR is not to completely bring the two in synch but to make sure the extra functionality we add to one makes it into the other one, hence I would consider it out of scope for this PR.